bevrand.multivuerse/src/views/PlaylistCreationPage.vue

```diff
@@ -2,79 +2,79 @@
     <div id="playlistCreationArea" class="playlistcreation">
         <navbar></navbar>
         <div id="creationArea" class="creationfield">
+
             <span class="label">
-                <a class="image featured" ><img :src="imageUrl" ></a><br>
-                <button v-on:click="setImage" class="submitbutton"> Change Image</button>
-                <a class="playlistselector" id="playlistName"><h3> Playlist: {{ displayName }}</h3>  </a>
-                <h3 style="margin-bottom: 0.5em">Beverages:</h3><br>
-                <div v-if="expandPopularCocktails">
+
+                <a class="playlistselector" id="playlistName"><h3 style="margin-top: 0.5em"> Playlistname: {{ displayName }}</h3></a>
+
+                <h3 style="margin-bottom: 0.5em" v-if="beverages.length === 0">Start adding beverages!</h3>
+                        <ul id="beverageList">
+            <li v-for="(beverage, index) in beverages" :key="index">
+                {{ beverage }}
+                <a class="hover-edit">
+                    <font-awesome-icon icon="pencil-alt" class="icons" v-on:click="editDrink(index)"/>
+                </a>
+                <a class="hover-delete">
+                    <font-awesome-icon icon="trash" class="icons" v-on:click="removeDrink(index)"/>
+                </a>
+            </li>
+        </ul>
+
+                <input v-model="drink" placeholder="Custom beverage" type="text" class="adddrink" value=""/>
+                <a class="hover-add">
+                    <font-awesome-icon v-on:click="addBeverage()" icon="plus-circle" size="lg"/></a>
+                <br>
+
+            <div v-if="expandPopularCocktails">
 
                 <button class="circledbutton"
                         v-for="(item, index) in popularCocktails"
                         :key="index"
                         style="background-color: orange"
                         v-on:click="() => setPopularDrink(item)">
                     <span>{{item}}</span>
-                </button>
```

Comment:
Alignment of closing button element seems to be not aligned with opening button element